sentry-uitest/sentry-uitest-android-benchmark/src/main/AndroidManifest.xml

```diff
@@ -0,0 +1,25 @@
+<?xml version="1.0" encoding="utf-8"?>
+<manifest xmlns:android="http://schemas.android.com/apk/res/android"
+    xmlns:tools="http://schemas.android.com/tools"
+    package="io.sentry.uitest.android.benchmark">
+
+    <!--
+      Important: disable debugging for accurate performance results
+
+      In a com.android.library project, this flag must be disabled from this
+      manifest, as it is not possible to override this flag from Gradle.
+    -->
+    <application
+      android:theme="@style/Theme.AppCompat"
+      android:label="Sentry Benchmark"
+      android:icon="@android:mipmap/sym_def_app_icon"
+      android:debuggable="false"
+      tools:ignore="HardcodedDebugMode">
+
+      <activity android:name=".BenchmarkActivity" />
+
+      <meta-data android:name="io.sentry.auto-init" android:value="false" />
```

Comment:
Maybe remove the content providers instead? https://docs.sentry.io/platforms/android/configuration/manual-init/ it's more efficient.